rcore/api_func_symbols.h

```diff
@@ -10,706 +11,706 @@ bool persist_exists(void) { return false; }
-void unalloc640() { printf("UNALLOC 640\n"); }
-void unalloc641() { printf("UNALLOC 641\n"); }
-void unalloc642() { printf("UNALLOC 642\n"); }
-void unalloc643() { printf("UNALLOC 643\n"); }
-void unalloc644() { printf("UNALLOC 644\n"); }
-void unalloc645() { printf("UNALLOC 645\n"); }
-void unalloc646() { printf("UNALLOC 646\n"); }
-void unalloc647() { printf("UNALLOC 647\n"); }
-void unalloc648() { printf("UNALLOC 648\n"); }
-void unalloc649() { printf("UNALLOC 649\n"); }
-void unalloc650() { printf("UNALLOC 650\n"); }
-void unalloc651() { printf("UNALLOC 651\n"); }
-void unalloc652() { printf("UNALLOC 652\n"); }
-void unalloc653() { printf("UNALLOC 653\n"); }
-void unalloc654() { printf("UNALLOC 654\n"); }
-void unalloc655() { printf("UNALLOC 655\n"); }
-void unalloc656() { printf("UNALLOC 656\n"); }
-void unalloc657() { printf("UNALLOC 657\n"); }
-void unalloc658() { printf("UNALLOC 658\n"); }
-void unalloc659() { printf("UNALLOC 659\n"); }
-void unalloc660() { printf("UNALLOC 660\n"); }
-void unalloc661() { printf("UNALLOC 661\n"); }
-void unalloc662() { printf("UNALLOC 662\n"); }
-void unalloc663() { printf("UNALLOC 663\n"); }
-void unalloc664() { printf("UNALLOC 664\n"); }
-void unalloc665() { printf("UNALLOC 665\n"); }
-void unalloc666() { printf("UNALLOC 666\n"); }
-void unalloc667() { printf("UNALLOC 667\n"); }
-void unalloc668() { printf("UNALLOC 668\n"); }
-void unalloc669() { printf("UNALLOC 669\n"); }
-void unalloc670() { printf("UNALLOC 670\n"); }
-void unalloc671() { printf("UNALLOC 671\n"); }
-void unalloc672() { printf("UNALLOC 672\n"); }
-void unalloc673() { printf("UNALLOC 673\n"); }
-void unalloc674() { printf("UNALLOC 674\n"); }
-void unalloc675() { printf("UNALLOC 675\n"); }
-void unalloc676() { printf("UNALLOC 676\n"); }
-void unalloc677() { printf("UNALLOC 677\n"); }
-void unalloc678() { printf("UNALLOC 678\n"); }
-void unalloc679() { printf("UNALLOC 679\n"); }
-void unalloc680() { printf("UNALLOC 680\n"); }
-void unalloc681() { printf("UNALLOC 681\n"); }
-void unalloc682() { printf("UNALLOC 682\n"); }
-void unalloc683() { printf("UNALLOC 683\n"); }
-void unalloc684() { printf("UNALLOC 684\n"); }
-void unalloc685() { printf("UNALLOC 685\n"); }
-void unalloc686() { printf("UNALLOC 686\n"); }
-void unalloc687() { printf("UNALLOC 687\n"); }
-void unalloc688() { printf("UNALLOC 688\n"); }
-void unalloc689() { printf("UNALLOC 689\n"); }
-void unalloc690() { printf("UNALLOC 690\n"); }
-void unalloc691() { printf("UNALLOC 691\n"); }
-void unalloc692() { printf("UNALLOC 692\n"); }
-void unalloc693() { printf("UNALLOC 693\n"); }
-void unalloc694() { printf("UNALLOC 694\n"); }
-void unalloc695() { printf("UNALLOC 695\n"); }
-void unalloc696() { printf("UNALLOC 696\n"); }
-void unalloc697() { printf("UNALLOC 697\n"); }
-void unalloc698() { printf("UNALLOC 698\n"); }
-void unalloc699() { printf("UNALLOC 699\n"); }
-void unalloc700() { printf("UNALLOC 700\n"); }
+void unalloc1() { SYS_LOG("apifs", APP_LOG_LEVEL_ERROR, "UNALLOC 1"); }
```

Comment:
I feel like `apifs` could be misleading, as there is no `fs`.
Maybe `api_symbols` or `api_table` might be a better name